en/0-intro.md

```diff
@@ -2,38 +2,33 @@
 
 ## Abstract
 
-Today’s work market lacks efficiency because of high transaction costs[^1][^2][^3] and frequent conflicts of interests between the employer and the employee.
+The current work market is inefficient due to high transaction costs and frequent conflicts of interest between employers and employees. 
 
-The WoM (World of Meeds) proposes a new paradigm for reducing transaction costs and reinventing working relationships. It bridges the gap between traditional companies and new forms of organisation by proposing a fair and transparent incentive system that realigns the interests of all parties.
+The World of Meeds (WoM) proposes a new paradigm to address these issues by reducing transaction costs and reinventing working relationships. It's a digital platform that bridges the gap between traditional companies and new forms of organization by introducing a fair and transparent incentive system that realigns the interests of all parties.
 
 ## Introduction
 
-The work market today is broken. It fails to satisfy both workers and employers.
+The current state of the work market fails to satisfy both workers and employers[^1][^2][^3]. 
 
-On the employee side, it is difficult to find the right jobs. The job market favors certain types of individuals, leaving great skill sets behind. This comes from a set of human biases – racial based, gender based, handicap based, introvert based etc. The work market also lacks transparency as to the available positions and employment conditions.
```

Comment:
Th term "employee" describes one who is WORKING for an organization, business firm or company. These set of people have found their jobs already. I prefer it should be replaced with the term "JOB SEEKERS"

Reply:
> The term "employee" describes one who is WORKING for an organization, business firm or company. These set of people have found their jobs already. I prefer it should be replaced with the term "JOB SEEKERS"

No. @Amby01  In this sentence, we're referring to the people already hired to do some work for the employer (organization). We're not referring to the problem of finding a job. The goal is this whole pull request is to expand from traditional employment model problems to expand to other forms of work relationship. Hence we later introduce  the term "contributor" to encompass :  employees, contractors, freelances,  independents contributors, or even volunteers